contracts/clearing/TradingApp.sol

```diff
@@ -67,21 +33,24 @@ contract TradingApp is IForceMoveApp {
 		}
 
 		bytes memory candidateData = candidate.variablePart.appData;
+
 		// settlement
-		// TODO: unsure whether we should check the proof when consensus is reached
-		if (candTurnNum % 2 == 0 && proof.length == 0) {
+		uint8 signaturesNum = NitroUtils.getClaimedSignersNum(candidate.signedBy);
+		if (candTurnNum % 2 == 0 && signaturesNum == 2 && proof.length > 0) {
```

Comment:
```suggestion
		if (candTurnNum % 2 == 0 && signaturesNum == 2 && proof.length >= 2) {
```

Reply:
Settlement requires at least 1 order-response pair

Reply:
> Unfortunately, we CAN'T state that if proof.length == 2, then candidate is Order, as Settlement can happen after just 1 order too.

Here by `after 1 order too` I meant the Order-Response pair. In case of a Settlement `[Order, OrderResponse]` (length 2) is a valid proof, and in case of an Order `[Order, OrderResponse]` is a valid proof too. I wanted to highlight that just the proof length alone is not enough to differentiate these 2 cases.